### DISCUSSION TITLE
Filtering a label column

### DISCUSSION THREAD

**Discussion by taalas** [rappasoft/laravel-livewire-tables#737] [Q&A]
Hi,

I am currently generating a column as a result of other columns data as a label (I analyze different dates and assign a value depending on multiple conditions). Since applying filters seems to only work at builder level (so directly on the db query). It doesn't seem to be possible to filter the table using a computed (label) value.

Ist this true? Is there anything I can to do work around this issue?

**Comment by taalas** [comment]
Alternatively I could filter an accessor, but this doesn't change the initial problem, right?

**Comment by rappasoft** [comment]
Can you filter using raw SQL? If so you can just override the default filter logic on the column.

**Comment by taalas** [comment]
Theoretically I could. But the computed values would needed to be replicated as SQL queries. So filtering the computed values would be much easier in this case. So this would be a filter on the result set, not in the source data.

**Comment by rappasoft** [comment]
Might be difficult since in the end the table is just one giant SQL query lol. Open to suggestions though.

**Comment by inmanturbo** [comment]
What about hooking into the query at the end? Sounds like maybe a post-query hook might work?

**Comment by lrljoe** [comment]
Is in both v2 and v3 now.
